app.js

```diff
@@ -1,25 +1,104 @@
-const express = require('express')
-const logger = require('morgan')
-const cors = require('cors')
+const express = require("express");
+const morgan = require("morgan");
+const cors = require("cors");
+const { connectDB } = require("./models/db");
+const bodyParser = require("body-parser");
 
-const contactsRouter = require('./routes/api/contacts')
+const {
+  listContacts,
+  getById,
+  addContact,
+  removeContact,
+  updateContact,
+  validateContact,
+} = require("./routes/api.js/contacts");
```

Comment:
```suggestion
} = require("./routes/api/contacts");
```
To extension nie powinno byc. W db.js masz jedno miejsce gdzie masz metody a tu importujesz z innego miejsca.